deposit/src/main/java/edu/unc/lib/deposit/transfer/TransferBinariesToStorageJob.java

```diff
@@ -103,6 +103,10 @@ private void transferBinaries(Resource resc, BinaryTransferSession transferSessi
             transferDepositManifests(objPid, resc, transferSession);
         }
 
+        // Add premis log location to deposit model
+        Path premisLogPath = getPremisFile(objPid).toPath();
```

Comment:
Could you verify that this is the right path? My memory is that this is the pre-ingest location where the premis file is stored, rather than post-ingest. I don't think the path of the post-ingest log exists until after IngestContentObjectJob and IngestDepositRecordJob have completed. 

The following might be a way to get the path in `RegisterToLongleafJob`, but i haven't tested it:
```
BinaryObject premisBin = repositoryObjectLoader.getBinaryObject(DatastreamPids.getMdEventsPid(pid));
URI premisStorageUri = premisBin.getContentUri();
```
But only after the ingest content/deposit jobs have completed, meaning it wouldn't work in this job since the objects haven't been ingested into fedora yet.